sematic/db/migrations/20230314141123_add_jobs_table.sql

```diff
@@ -0,0 +1,22 @@
+-- migrate:up
+
+CREATE TABLE jobs (
+    name TEXT NOT NULL,
+    namespace TEXT NOT NULL,
```

Comment:
Please see [this comment](https://github.com/sematic-ai/sematic/pull/668#discussion_r1137513386).

Reply:
These are both part of the primary key, which Postgres automatically indexes